docusaurus/docs/iOS/guides/working-with-messagelist-layout.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Working with MessageList Layout
+---
+
+## `ChatMessageLayoutOptionsResolver`
+
+Almost anything related to the layout and appearance of the message cell can be done by subclassing [`ChatMessageLayoutOptionsResolver`](../ReferenceDocs/Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.md). 
+On  `ChatMessageLayoutOptionsResolver` there is intance property `options`([ChatMessageLayoutOptions](../ReferenceDocs/Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptions.md)) which holds available options for the cell. 
```

Comment:
We should put more emphasize on `ChatMessageLayoutOptions`, like

> For more information about available layout options, please see ...

Reply:
still valid

Reply:
Thanks and sorry, didn't notice... Added.